tinytga/src/embedded_graphics.rs

```diff
@@ -0,0 +1,56 @@
+use crate::{parse_error::ParseError, ImageOrigin, TgaRaw};
+use core::marker::PhantomData;
+use embedded_graphics::prelude::*;
+
+/// TGA image for use with embedded-graphics
+#[derive(Debug)]
+pub struct Tga<'a, C> {
+    tga: TgaRaw<'a>,
+    color_type: PhantomData<C>,
+}
+
+impl<'a, C> Tga<'a, C> {
+    /// Parse a TGA image from a byte slice
+    pub fn from_slice(data: &'a [u8]) -> Result<Self, ParseError> {
+        Ok(Self {
+            tga: TgaRaw::from_slice(data)?,
+            color_type: PhantomData,
+        })
+    }
+}
+
+impl<C> ImageDrawable for Tga<'_, C>
+where
+    C: PixelColor + From<<C as PixelColor>::Raw>,
+{
+    type Color = C;
+
+    fn draw<D>(&self, target: &mut D) -> Result<(), D::Error>
+    where
+        D: DrawTarget<Color = C>,
+    {
+        // TGA files with the origin in the top left corner can be drawn using `fill_contiguous`.
+        // All other origins are drawn by falling back to `draw_iter`.
```

Comment:
Would this be worth mentioning in a `# Performance` section in the `Tga` struct docs?

Reply:
That's a good idea! I've added a performance section to the docs.